src/list/manage/components/promote-banner.css

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright (c) 2019 Matthew A. Miller
+ * 
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+.promotion {
+    min-width: max-content;
+    display: flex;
+    flex-flow: row nowrap;
+    align-items: center;
+    justify-content: baseline;
+}
+.promotion .content {
```

Comment:
You might want to set a max-width on the `.content` div, so that the detail text wraps nicely in German and other languages that tend to have lengthy translations

Reply:
I put a max-width on the overall banner, but max-width here would keep other layout in a better state.